sdk/src/main/java/com/uid2/EUIDManager.kt

```diff
@@ -0,0 +1,107 @@
+package com.uid2
+
+import android.content.Context
+import com.uid2.UID2Manager.Companion
+import com.uid2.UID2Manager.Companion.APPLICATION_ID_DEFAULT
+import com.uid2.UID2Manager.Companion.UID2_API_URL_DEFAULT
+import com.uid2.network.DefaultNetworkSession
+import com.uid2.network.NetworkSession
+import com.uid2.storage.FileStorageManager
+import com.uid2.storage.FileStorageManager.Store.EUID
+import com.uid2.storage.StorageManager
+import com.uid2.utils.InputUtils
+import com.uid2.utils.Logger
+import com.uid2.utils.TimeUtils
+import kotlinx.coroutines.Dispatchers
+
+public class EUIDManager {
+
+    public companion object {
+        public data class Environment private constructor(
```

Comment:
In iOS, you implemented a single `Environment` type that contained both UID2 vs EUID environments. However, here, you are having two separate/dedicated types per UID2 vs EUID. I probably prefer this approach, as it makes it much more explicit and less likely for people to make a mistake, but why?